manifests/4.6/sriov-network-operator-sriovnetworknodestate.crd.yaml

```diff
@@ -57,15 +57,10 @@ spec:
                           type: string
                         vfRange:
                           type: string
-                      required:
-                      - policyName
-                      - resourceName
                       type: object
                     type: array
                 required:
-                - name
```

Comment:
In which case, a interface doesn't have a name?

Reply:
> > OLM doesn't allow CRD to introduce new required fields during an upgrade.
> 
> Could you link to the OLM requirement?
> If these required options are removed from CRD, will they be admitted in operator webhook?

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#on-compatibility

You can also find more detail in the BZ comments.

Reply:
SriovNetworkNodeState CRs are managed by the operator itself. So we shall not worry about the illegal configuration too much.